health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/AppConfigurationParentLayer.js

```
@@ -214,17 +221,18 @@ const AppConfigurationParentRedesign = ({ formData = null, isNextTabAvailable, i
-        ? {
-            ...parentState?.actualTemplate?.actualTemplate,
-            version: parentState?.actualTemplate?.version + 1,
-            pages: reverseData,
-          }
-        : {
-            ...parentState?.actualTemplate,
-            version: parentState?.actualTemplate?.version + 1,
-            pages: reverseData,
-          };
+      const reverseFormat =
+        cacheData && cacheData?.data?.data
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)